core/src/main/java/com/datastax/oss/driver/internal/core/type/codec/TupleCodec.java

```diff
@@ -186,15 +187,25 @@ public TupleValue parse(@Nullable String value) {
       } catch (IllegalArgumentException e) {
         throw new IllegalArgumentException(
             String.format(
-                "Cannot parse tuple value from \"%s\", invalid CQL value at character %d",
-                value, position),
+                "Cannot parse tuple value from \"%s\", invalid CQL value at character %d: %s",
+                value, position, e.getMessage()),
```

Comment:
Shouldn't we throw the IllegalArgumentException with suppressed exception and pass `e` to it?

Reply:
But we are passing `e` as the cause already.